### DISCUSSION TITLE
Save where you left off after closing the tab

### DISCUSSION THREAD

**Discussion by zenVentzi** [monkeytypegame/monkeytype#700] [Ideas] [answered]
Feature request. 

E.g. I've pasted a long text, I've come to somewhere in the middle but I close the tab and want to continue where I left off few hours later.

**Comment by frogfrogfrogfrogfrog** [comment]
This is a cool idea to be able to go back and type long texts but I don't think it should be executed as saving whenever you leave the tab. I think a better idea would be able to have some kind of pause option for long tests.

**Reply by ryan-harrington** [reply]
Being able to pause during tests could give an unfair advantage over other typists who don't pause.

**Reply by frogfrogfrogfrogfrog** [reply]
could you elaborate an example where this could give an unfair advantage? The only one that I could think of is with long tests but pause time could be considered afk time

**Reply by ryan-harrington** [reply]
It's easy to burst for a few seconds, pause, and repeat.

**Reply by frogfrogfrogfrogfrog** [reply]
that can still be solved with pause time being considered afk time where if your paused it counts as being afk 

there could be a third wpm counter that takes that into account

I do agree though that a pause feature isn't needed but I don't agree with that it could give typists an unfair advantage.

**Reply by ryan-harrington** [reply]
What happens when pause time is considered as AFK time?

**Reply by frogfrogfrogfrogfrog** [reply]
actually scratch that idea

that would defeat the whole point of pause time

I'm just gonna 100% agree with you that a pause feature wouldn't work.

**Comment by Miodec** [comment, marked as answer]
As others have mentioned in the thread - not a good idea for challenges and abusing in general to get higher scores.

EDIT: Nevermind, I didn't read the actual question - I'm not sure how I would go about checking the state of the long test to be honest.
